be/src/olap/wal/wal_table.cpp

```diff
@@ -89,12 +89,17 @@ Status WalTable::_relay_wal_one_by_one() {
         wal_info->add_retry_num();
         auto st = _replay_wal_internal(wal_info->get_wal_path());
         auto msg = st.msg();
+        std::filesystem::path file_path(wal_info->get_wal_path());
+        if (!std::filesystem::exists(file_path)) {
+            LOG(WARNING) << "wal file=" << wal_info->get_wal_path() << " does not exist";
+        }
+        auto file_size = std::filesystem::file_size(file_path);
```

Comment:
if the file does not exist, file_size maybe throw a exception, and we should catch it.